docs/tutorials/maps.ipynb

```diff
@@ -10,15 +10,16 @@
     "\n",
```

Comment:
<p>It seems the formatting of the text  is odd here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
It's fine to skip a line here no ?